.dockerignore

```diff
@@ -0,0 +1,25 @@
+# Docker volume
+docker-volume
+
+# pytest
+.pytest_cache
+
+# firefox directories
+firefox-bin
+*.app
+
+# PyCharm
+.idea
+
+# Local Python virtual environments
+venv
+
+# npm packages
```

Comment:
Hi, I would suggest to keep the _node_modules_ since I'm currently working on a Dockerfile-dev, which includes these files. The reason for that is that with the Dockerfile-dev, you should get a container that allows you to easily modify and rebuild OpenWPM, so the _node_modules_ are definitely useful for that.

I also created a PR for that: #332

Reply:
The main reason of skipping node_modules is however not performance-related, but compatibility. Node modules many times contain binary builds that are compiled for the current OS, and sending node_modules from a Mac/Windows machine into the Docker context for building a Ubuntu image is a recipe for weirdness :)